lib/rodauth/features/webauthn.rb

```diff
@@ -464,7 +459,7 @@ def webauthn_keys_ds
 
     def webauthn_auth_credential_from_form_submission
       begin
-        webauthn_credential = WebAuthn::Credential.from_get(webauthn_auth_data)
+        webauthn_credential = WebAuthn::Credential.from_get(webauthn_auth_data, relying_party: relying_party)
```

Comment:
I decided to go with this approach because using `WebAuthn:RelyingParty` seemed a little bit ugly, but I’ll leave it here just in case:

```diff
    diff --git a/lib/rodauth/features/webauthn.rb b/lib/rodauth/features/webauthn.rb
index 9af3cd3..52e6eda 100644
--- a/lib/rodauth/features/webauthn.rb
+++ b/lib/rodauth/features/webauthn.rb
@@ -356,18 +356,18 @@ module Rodauth
       nil
     end

-    def valid_webauthn_credential_auth?(webauthn_credential)
-      ds = webauthn_keys_ds.where(webauthn_keys_webauthn_id_column => webauthn_credential.id)
+    def valid_webauthn_credential_auth?(webauthn_auth_data)
+      ds = webauthn_keys_ds.where(webauthn_keys_webauthn_id_column => webauthn_auth_data[“id”])
       pub_key, sign_count = ds.get([webauthn_keys_public_key_column, webauthn_keys_sign_count_column])

       (challenge = param_or_nil(webauthn_auth_challenge_param)) &&
         (hmac = param_or_nil(webauthn_auth_challenge_hmac_param)) &&
         (timing_safe_eql?(compute_hmac(challenge), hmac) || (hmac_secret_rotation? && timing_safe_eql?(compute_old_hmac(challenge), hmac))) &&
-        webauthn_credential.verify(challenge, public_key: pub_key, sign_count: sign_count) &&
+        (webauthn_credential = relying_party.verify_authentication(webauthn_auth_data, challenge, public_key: pub_key, sign_count: sign_count)) &&
         ds.update(
           webauthn_keys_sign_count_column => Integer(webauthn_credential.sign_count),
           webauthn_keys_last_use_column => Sequel::CURRENT_TIMESTAMP
-        ) == 1
+        ) == 1 && webauthn_credential
     end

     def remove_webauthn_key(webauthn_id)
@@ -459,8 +459,7 @@ module Rodauth

     def webauthn_auth_credential_from_form_submission
       begin
-        webauthn_credential = WebAuthn::Credential.from_get(webauthn_auth_data, relying_party: relying_party)
-        unless valid_webauthn_credential_auth?(webauthn_credential)
+        unless webauthn_credential = valid_webauthn_credential_auth?(webauthn_auth_data)
           throw_error_reason(:invalid_webauthn_auth_param, invalid_key_error_status, webauthn_auth_param, webauthn_invalid_auth_param_message)
         end
       rescue WebAuthn::SignCountVerificationError
```